project 1/index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>The Crane Wife</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Libre+Baskerville:400,700&display=swap" rel="stylesheet">
```

Comment:
In your webfont stylesheet, you call for font weights 400 and 700, but your stylesheet uses the following weights: 100, 400, 500, 600

To get the most benefit out of your webfont, make sure you call all the weights used in your styles.